### PR TITLE
fix: hide free sticker notice when stickers are added to the shopping cart

### LINKED ISSUE #246: Free Gift bug - It shows a free sticker pack with stickers
If you try to order a pack of stickers - The site shows in the cart "We will add the Gatsby Sticker Pack as a FREE bonus to your order!" I don't know if that's the intention...

### PR DESCRIPTION
**Pull request by KirankumarAmbati**
closes #246

### CODE REVIEW

File: src/components/Cart/Cart.js [hunk 1]
```diff
@@ -340,6 +341,12 @@ class Cart extends Component {
             (total, item) => total + item.quantity,
             0
           );
+          
+          checkout.lineItems.forEach(({title}) => {
+            if(title === 'Gatsby Sticker Pack') {
```
Comment (jlengstorf) [inline src/components/Cart/Cart.js]: I think we may want to check for something other than the title — there's no guarantee we won't tweak the name of the product. Something like the slug or ID would probably be better here.

Comment (KirankumarAmbati) [reply src/components/Cart/Cart.js]: @jlengstorf  Gatsby Sticker Pack ID is having 140 characters length. Is it okay to use it ?

Comment (jlengstorf) [reply src/components/Cart/Cart.js]: That should be fine. Thanks!

Comment (KirankumarAmbati) [reply src/components/Cart/Cart.js]: @jlengstorf Made changes to replace `title` with `ID`. Please can you verify. Thanks!

**Pull request merged**